react/Installments.tsx

```diff
@@ -14,79 +12,31 @@ interface Installment {
   NumberOfInstallments: number
 }
 
-const CSS_HANDLES = [
-  'installments',
-  'installmentsNumber',
-  'installmentValue',
-  'installmentsTotalValue',
-  'interestRate',
-] as const
-
 const Installments: StorefrontFC<BasicPriceProps> = props => {
   const { message, markers } = props
-  const handles = useCssHandles(CSS_HANDLES)
   const { selectedItem } = useContext(ProductContext)
-
-  const commercialOffer = selectedItem?.sellers[0]?.commertialOffer
+  const commertialOffer = selectedItem?.sellers[0]?.commertialOffer
```

Comment:
I just fixed a typo here

Reply:
No, you've applied the typo again 😆 the right one is `commercialOffer`

Reply:
wat